tools/radar/capture_radar.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+require File.expand_path("../../config/environment", __dir__)
+require 'trollop'
+require './tools/radar/rollup_radar_mixin'
+include RollupRadarMixin
+
+opts = Trollop.options(ARGV) do
+  banner "USAGE:   #{__FILE__} -h <number of hours back to query metrics>\n" \
+         "Example: #{__FILE__} -d <number of days back to query metrics>"
+
+  opt :hours, "Hours", :short => "h", :type => :int, :default => 2
```

Comment:
@gtanzillo  so given it's time T now

<T-4h, T-3h) - purging should start soon, so we might be missing some metrics since they are being purged in batches
<T-3h, T-2h) - on a reasonable sized env, this has the highest probability of having all metrics of all containers
<T-2h, T-1h) - lower probability of missing metrics, since refresh haven't finished (new containers haven't been schedule for metrics collection) and metrics collections haven't finished
<T-1h, T) - high probability of missing  metrics, since refresh haven't finished (new containers haven't been schedule for metrics collection) and metrics collections haven't finished